docs/user_guide/fl_simulator.rst

```diff
@@ -4,13 +4,16 @@
 NVIDIA FLARE FL Simulator
 #########################
 
-The NVIDIA FLARE FL Simulator was added in version 2.2 to help researchers accelerate the development of federated
+The NVIDIA FLARE FL Simulator was added in version 2.2 to help researchers
+accelerate the development of federated
```

Comment:
This spacing doesn't end up making it into the html, so it doesn't matter except for display on the screen. There isn't a formatter for this so it can be somewhat arbitrary.